### ISSUE TITLE
Throttles rolling back a few seconds after takeoff

### ISSUE THREAD

**Issue by JC10255** [Heavy-Division/B78XH#194]
Hello. I’m having an issue with the heavy MOD on the 787. After takeoff, about 300 feet in the air, the throttles are rolling back to around 60% N1, and then slowly coming back to 88% or so that represents TO-1. At that exact time, the speed I have dialed in (V2) disappears in the MCP window and the A/T light goes out. Obviously, this causes a dangerous arresting of my climb rate at a crucial time.

I set up everything at the gate in the FMS appropriately.  TO-1 and CLB-1 settings with ambient temperature set. The autothrottle switches are turned on during my preflight and I also arm LNAV/VNAV before takeoff. In other words, everything is set as per real world procedures as closest possible.

When lined up with the runway, I move both throttles up to ensure they come up even and then press the black TOGA button on the TL’s directly below the white throttle handles. The TL’s come up as they should on their own and set TO-1 power. Then move my joystick throttles forward to match.

I rotate at VR and follow the FD bars which usually command 10 degrees of pitch or so. 

Positive rate, gear up.  

At about 2-300 feet, the A/T light turns off, the speed in the MCP window goes blank, my thrust rolls back to a dangerously low 60% or so (commanding the FD bars down to the horizon) and then the engines start to slowly come back up to 88%, but not before the aircraft comes dangerously close to the ground. 

It’s also important to note that in the engine instruments section of the center screen, where the green thrust mode annunciator should be showing “TO-1” and then CLB at 1500AGL, there are just two small green dashes.  The green dashes exist from the time I am back at the gate all the way through climb out and then for the duration of the flight. I can never get that section to show TO-1 or CLB.  I’m not sure if that’s related to the throttles rolling back.

Any help in this matter would greatly be appreciated. I hope it’s something that I’m doing wrong and not a legitimate bug .

**Comment by Ash624** [comment]
Is the aircraft going into VNAV speed? Have you commanded autopilot at this stage?
When the MCP speed window goes blank, its going into VNAV speed (the auto thrust light will go blank too) What speed are you setting for climb speed in the window (typically V2+10kts ish)?
I dont engage autopilot until around 2000ft. I set climb speed in the window, arm VNAV and LNAV before rolling, but not auto thrust. I then TOGA on the runway using the button/s on the thrust levers (and move my actual thrust levers forward to 75% approx) The auto thrust will engage (TOGA). The auto thrust should fly your pre-set speed (it often overshoots a bit before settling back) You can then turn A/P on and it should go into VNAV/LNAV and managed speed. The autothrust light will go out until at cruise. You must remember to move your own actual thrust levers forward or the sim ones will roll back (as you have stated you do)
Are you setting a climb speed in the window?

**Comment by Ash624** [comment]
Hi,
Sounds like you are doing everything right. What speed are you putting into the MCP speed window?
It should typically be V2+10kts (ish).
Are you completing the necessary FMGC inputs, including thrust settings, take off speeds etc?
I turn on VNAV/LNAV prior to rolling (at taxi), but not autothrust. When on the runway, like you, I press the button/s on the thrust levers, then autothrust indicator will light up on the MCP, thrust levers will advance by themselves. You **MUST** move your own thrust lever/s way forward, maybe even to the full extent, or the 787 ones will spool back shortly after take off. It should then climb at your selected speed (although it usually overshoots a bit prior to settling down)
I usually fly the very first part of the departure by hand (using autothrust only) and then select autopilot around 2000ft. If the speed dialled into the MCP window goes blank, as well as the A/T light going out, it has gone into VNAV speed mode. It will do all of this once autopilot has been selected, and should then fly the VNAV speeds until cruise is reached. The autothrust light will then relight at selected cruise level. You can force a "selected" speed again at anytime by pressing the speed select button/dial, this then re-lights/shows the speed in the box for you to adjust.
Give it a try, hope it helps.

**Comment by JC10255** [comment]
Thank you for your reply. I think my issue can be best summed up by watching what happens in this video right after takeoff when his throttles went to idle. And this was while receiving direct instruction from a RL 787 pilot. 

Even though this video is a few months old and the heavy mod went through a few updates, the issue is still occurring as it does in the video below.

The only way I can get thrust to maintain its integrity during second segment climb without rolling back is to select my Accel and thrust reduction altitude both the 400ft in the FMS. Then the problem disappears. The only drawback with that method is, to get my required claim gradient, I have to ignore the horizontal FD bars

https://youtu.be/xg853BCWFiI

Jason

**Comment by Ash624** [comment]
Jason,
I am not sure I can add anything else unfortunately. I have not been flying the 787 HD mod for that long, I have spent the last 18 months accumulating around 1200hrs on the FBW a32nx, but decided to start doing some long haul on Vatsim. I only have around 50 hrs on the 787, I am no Boeing expert, so others may be more suited to advise you further.  I did get thrust reduction/rollback on my first few flights (like the video), but I solved this by moving my own hardware levers further forward on the take off roll.
I have had no further rollback after take off.
With all v speeds setup correctly before departure, and LNAV/VNAV armed, I look down at the thrust levers, press the button on the levers to command acceleration, the thrust levers move forward, I then move my own hardware levers to Flex position (I have the Airbus Thrustmaster quadrant setup). I do not touch my own hardware thrust levers again during the entire flight until the retard for landing. I control everything using the MCP panel. They stay in flex position.
At rotate speed I pitch for around 10 degree's, retract the gear, climb out. The 787 usually accelerates slightly beyond the V2+ speed, then reduces to my selected speed (approx). I fly the SID by hand at first, then when ready I switch on autopilot, usually around 2000ft or so. The 787 autothrust light goes out, the selected speed goes blank, and it continues in VNAV/LNAV mode. I select flaps up during the acceleration phase as it speeds up towards 250kts.
Sorry I cant help you further, but good luck!!
